docs/source/markdown/podman-image-diff.1.md

```diff
@@ -1,30 +1,45 @@
 % podman-image-diff(1)
 
 ## NAME
+
 podman-image-diff - Inspect changes on an image's filesystem
 
 ## SYNOPSIS
+
 **podman image diff** [*options*] *name*
 
 ## DESCRIPTION
-Displays changes on a container or image's filesystem.  The container or image will be compared to its parent layer
+
+**podman image diff** displays changes on an image's filesystem.  The image will be compared to its parent layer.
+
+The output is prefixed with the following symbols:
+
+| Symbol | Description |
+|--------|-------------|
+| A | A file or directory was added. |
+| D | A file or directory was deleted. |
+| C | A file or directory was changed. |
 
 ## OPTIONS
 
 #### **--format**
```

Comment:
Should be `**--format**=*format*`. Maybe adding a `-f` option here would be also a good idea, but i believe you do not have to do this. But it is worth mentioning.

`JSON` should be written in small letters, since it does not work when written in caps lock. So the old version was correct.

If no there is no input behind format the contents are simply listed below each other in the file. So it is not really true that `json` is the only valid option.

Reply:
I think acronyms are supposed to be capitalized and if you are talking about the man pages then it does work just fine? 
![Screen Shot 2021-06-09 at 10 15 04 AM](https://user-images.githubusercontent.com/65443164/121295291-6539a800-c90c-11eb-96c9-cc070dd06e5c.png)

Reply:
The bold version after an eqaul sign should be used when there is a standard input. Since --format does not have this it should be italic.

Reply:
I'll fix that, can you confirm the JSON part? Because I still think acronyms should be in capitals